test/test_torch.py

```diff
@@ -5422,6 +5422,70 @@ def test_set_flush_denormal(self):
         self.assertEqual(double_tensor[2], 0.0, prec=0.0)  # tiny_double to zero
         torch.set_flush_denormal(False)
 
+    def test_unique_cpu(self):
+        x = torch.LongTensor([1, 2, 3, 2, 8, 5, 2, 3])
+        expected_unique = torch.LongTensor([1, 2, 3, 5, 8])
+        expected_inverse = torch.LongTensor([0, 1, 2, 1, 4, 3, 1, 2])
+        empty_inverse = torch.LongTensor([])
+
+        x_unique, x_inverse = torch.unique(x)
+        self.assertEqual(
+            expected_unique.tolist(), sorted(x_unique.tolist()))
+        self.assertEqual(empty_inverse, x_inverse)
+
+        x_unique, x_inverse = x.unique(return_inverse=True)
```

Comment:
If we want to be consistent with numpy we should only return the inverse if the user asked for it. Not a great design decision, but I don't think it's a good idea to diverge when we're using the same name

Reply:
TensorFlow's unique op also returns the inverse (https://www.tensorflow.org/api_docs/python/tf/unique), and same for Caffe2's (https://caffe2.ai/docs/operators-catalogue.html#unique). I'm happy to change any arg name to something else if you prefer, but I don't think it's worth diverging from the design decision convention if consistency is your main concern.